### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by aniseprakash**
Added few translations

### CODE REVIEW

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -86,7 +86,7 @@ msgstr ""
 
 msgctxt "install-duckduckgo"
 msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
-msgstr ""
+msgstr "%sஅழுத்தவும் %அனுமதிக்கவும்%அழுத்தவும் %சேர்க்கவும்%சரிபார்த்துக்கொள்ளவும்%s, பிறகு ஓகேவை அழுத்தவும்%s"
```
Comment (GioSensation) [inline locales/ta_IN/LC_MESSAGES/duckduckgo.po:89]: It looks like a the placeholder tags got messed in here. Would you mind checking it again? I include a possible suggestion below, but I don't know the language so you'd better double check.

```suggestion
msgstr "%sஅழுத்தவும் %sஅனுமதிக்கவும்%sஅழுத்தவும் %sசேர்க்கவும்%sசரிபார்த்துக்கொள்ளவும்%s, பிறகு %sஓகேவை%s அழுத்தவும்"
```

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -98,22 +98,22 @@ msgctxt "noresults"
 msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
-msgstr ""
+msgstr "இந்த தேடலுக்கு விடைகள் வந்திருக்க வேண்டும் என்றால் %sஇங்கே சொடுக்கவும்%s"
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo!%s"
-msgstr ""
+msgstr "%sடக்டக்கோவை பிறபலப்படுத்த உதவுங்கள்!%s"
```
Comment (GioSensation) [inline locales/ta_IN/LC_MESSAGES/duckduckgo.po:105]: Please, leave the brand name in English and in latin alphabet.

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -98,22 +98,22 @@ msgctxt "noresults"
...
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo%s."
-msgstr ""
+msgstr "%sடக்டக்கோவை பிறபலப்படுத்த உதவுங்கள்%s"
```
Comment (GioSensation) [inline locales/ta_IN/LC_MESSAGES/duckduckgo.po:109]: Please, leave the brand name in English and in latin alphabet.

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -163,43 +163,43 @@ msgid "About DuckDuckGo"
...
 msgid "Ad is suspicious"
-msgstr ""
+msgstr "சந்தேகத்திற்குரிய விளம்பரம்"
 
 #  Marked fuzzy because of translated trademark
 #. From Home page, options to "Add DuckDuckGo" to browser or set as home page.
 #, fuzzy
 msgid "Add DuckDuckGo"
-msgstr "சேர் கேலன் இலிருந்து வழிமாற்றப்பட்டது"
+msgstr "டக்டக்கோவை சேர்க"
```
Comment (GioSensation) [inline locales/ta_IN/LC_MESSAGES/duckduckgo.po:196]: Please, leave the brand name in English and in latin alphabet.

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -163,43 +163,43 @@ msgid "About DuckDuckGo"
...
 
 msgid "Add DuckDuckGo as a search engine"
-msgstr ""
+msgstr "டக்டக்கோவை உங்கள் முதன்மை தேடுபொறியாக அமைக்க"
```
Comment (GioSensation) [inline locales/ta_IN/LC_MESSAGES/duckduckgo.po:199]: Please, leave the brand name in English and in latin alphabet.

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -163,43 +163,43 @@ msgid "About DuckDuckGo"
...
 
 msgid "Add DuckDuckGo to %s"
-msgstr ""
+msgstr "டக்டக்கோவை %sல் சேர்க"
```
Comment (GioSensation) [inline locales/ta_IN/LC_MESSAGES/duckduckgo.po:202]: Please, leave the brand name in English and in latin alphabet.

**Comment by GioSensation** [comment]
Thanks for your contribution, @aniseprakash. I've left a few comments regarding translating the brand name and a placeholder mismatch. Please review and let me know. Everything else looks good. Thanks!

File: locales/ta_IN/LC_MESSAGES/duckduckgo.po (hunk 6 above)
Comment (aniseprakash) [reply locales/ta_IN/LC_MESSAGES/duckduckgo.po:202]: Sure will do

**Comment by GioSensation** [comment]
I'm closing this PR, as these changes are included in #374. Thanks.

**Pull request closed without merging**